lib/Notification/Notifier.php

```diff
@@ -563,7 +563,7 @@ protected function parseCall(INotification $notification, Room $room, IL10N $l):
 			if ($user instanceof IUser) {
 				if ($this->notificationManager->isPreparingPushNotification() || $room->hasSessionsInCall()) {
```

Comment:
this will create a conflict with #4324 so lets delay this until the other one is in, to avoid tones of rebases

Reply:
the PR in question was merged, no conflict has appeared :-D

Reply:
Yeah, it is 2 lines apart and that seems to be enough...